pkg/kubectl/rollback.go

```diff
@@ -221,9 +224,10 @@ func (r *DaemonSetRollbacker) Rollback(obj runtime.Object, updatedAnnotations ma
 	if !ok {
 		return "", fmt.Errorf("passed object is not a DaemonSet: %#v", obj)
 	}
-	versionedExtensionsClient := versionedExtensionsClientV1beta1(r.c)
 	versionedAppsClient := versionedAppsClientV1beta1(r.c)
-	versionedDS, allHistory, err := controlledHistories(versionedExtensionsClient, versionedAppsClient, ds.Namespace, ds.Name)
+	versionedExtensionsClient := versionedExtensionsClientV1beta1(r.c)
+	versionedObj, allHistory, err := controlledHistories(versionedAppsClient, versionedExtensionsClient, ds.Namespace, ds.Name, "DaemonSet")
+	versionedDS := versionedObj.(*externalextensions.DaemonSet)
```

Comment:
This will panic if `versionedObj` is not a `*externalextensions.DaemonSet`. It's probably okay, but it'll be safer to do something like: 
```go
versionedDS, ok := versionedObj.(*externalextensions.DaemonSet)
// return an error if !ok
```

Same comment for the other type assertion in this PR